### PR TITLE
ACS Model change for container ordering

### PR DESCRIPTION
**Pull request by ubhattacharjya**
### Summary

ACS Model change for container ordering
### Testing

New tests cover the changes: 

### Licensing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: agent/acs/model/api/api-2.json [hunk 1]
```diff
@@ -455,6 +455,18 @@
       "type":"list",
       "member":{"shape":"PortMapping"}
     },
+    "ProxyConfiguration":{
+      "type":"structure",
+      "members":{
+        "type":{"shape":"ProxyConfigurationType"},
+        "containerName":{"shape":"String"},
+        "properties":{"shape":"StringMap"}
+      }
```
Comment (yumex93) [inline agent/acs/model/api/api-2.json]: Please only includes changes that related to ur feature. Normally we only update changes related to our own features when we update acs model.

**Comment by suneyz** [comment]
please take a look at why integ and functional tests are failing

**Comment by petderek** [comment]
I'm investigating why windows tests are failing.  It looks like the dependency tests are failing and it isn't clear why

**Comment by petderek** [comment]
We've isolated the reason for windows tests failing: the windows containers are too resource constrained. We will open a seperate PR to address this.

**Pull request merged**